modules/bcftools/concat/main.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BCFTOOLS_CONCAT {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::bcftools=1.11" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bcftools:1.11--h7c999a4_0"
+    } else {
+        container "quay.io/biocontainers/bcftools:1.11--h7c999a4_0"
+    }
+
+    input:
+    tuple val(meta), path(vcfs)
+
+    output:
+    tuple val(meta), path("*.gz"), emit: vcf
+    path  "*.version.txt"        , emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    prefix       = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    bcftools concat \\
+        --output ${prefix}.vcf.gz \\
+        $options.args \\
```

Comment:
`--threads` is offered by bcftools for compression, so I would recommend adding this as per guidelines: http://samtools.github.io/bcftools/bcftools.html#common_options `--threads`)

Reply:
I was also in doubt regarding this one, I referred to other modules in `bcftools` but it seems that none was using this option and I imagined that there is some other global config that is intended to control that (apart from `process_medium` etc) . 

I'd be happy to add `--threads` here and update others as well - what do you think?

Reply:
I think it would be worth it given the guidelines also says that all output should be compressed by default anyway (and is the only case when `--threads` can be applied according to the docs)!

Reply:
Yup, done now :)